.gitignore

```diff
@@ -17,5 +17,5 @@ bridge/ffi/src/c_example/libdsnp_graph_sdk_ffi.a
 bridge/ffi/src/c_example/main
 bridge/ffi/src/c_example/dsnp_graph_sdk_ffi.h
 *.class
-bridge/node/dsnp_graph_sdk.node
-bridge/node/js/dsnp_graph_sdk.node
+bridge/node/grapsdk.node
+bridge/node/js/grapsdk.node
```

Comment:
Why the name change? Don't most of our other crates/packages for GraphSDK use the "dsnp-graph-*" pattern?